scripts/src/java/org/oppia/android/scripts/ci/BUILD.bazel

```
@@ -0,0 +1,18 @@
+"""
+Libraries corresponding to developer scripts that help with continuous integration workflows.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
+kt_jvm_library(
+    name = "compute_affected_tests_lib",
+    testonly = True,
+    srcs = [
+        "ComputeAffectedTests.kt",
+    ],
+    visibility = ["//scripts:oppia_script_binary_visibility"],
+    deps = [
+      "//scripts/src/java/org/oppia/android/scripts/common:bazel_client",
+      "//scripts/src/java/org/oppia/android/scripts/common:git_client",
```

```suggestion
        "//scripts/src/java/org/oppia/android/scripts/common:bazel_client",
        "//scripts/src/java/org/oppia/android/scripts/common:git_client",
```

This will fix the Bazel lint error

---

Done.